xwiki-platform-core/xwiki-platform-like/xwiki-platform-like-test/xwiki-platform-like-test-docker/src/test/it/org/xwiki/like/test/ui/LikeIT.java

```diff
@@ -129,4 +132,28 @@ void likeUnlikeDefaultConfiguration(TestUtils testUtils, TestReference testRefer
         assertTrue(likeButton.isDisplayed());
         assertEquals(1, likeButton.getLikeNumber());
     }
+
+    @Test
+    @Order(3)
+    void userProfileUIXLiveData(TestUtils testUtils, TestReference testReference)
+    {
+        testUtils.login(USER1, USER1);
+
+        // Re-creates and likes a page.
+        testUtils.deletePage(testReference);
+        testUtils.createPage(testReference, "", "");
+        LikeButton likeButton = new LikeButton();
+        likeButton.clickToLike();
+        assertEquals(1, likeButton.getLikeNumber());
```

Comment:
Why is this needed? We're supposed to be in a scenario and it would be weird that we don't already like a page in LikeIT elsewhere.

Reply:
At this point in the test user1 has 0 pages likes. I could indeed use user2 instead, which already should have a liked page.

Reply:
yes, in the same test and not in a new `@Test`.